1.1/spec/09-Part-06.adoc

```diff
@@ -32,6 +32,25 @@ geo:SpatialObject a rdfs:Class,
 |`/req/core/spatial-object-class`
 |===
 
+===== Class: geo:SpatialMeasure
+
+The class `geo:SpatialMeasure` is defined by the following:
+
+```
+geo:SpatialMeasure a rdfs:Class, 
+                    owl:Class;
+    rdfs:isDefinedBy <http://www.opengis.net/spec/geosparql/1.1>;
+    rdfs:label "Spatial Measure"@en;
+    rdfs:comment "The class represents a measurement of a feature though only 
+                spatial measurements. It is therefore naturally a subclass 
```

Comment:
I disagree that SpatialMeasure is a _subclass_ of spatial object. It behaves like a data-type, not a class of objects in the world.

Reply:
This is an error: the subclassing is removed in the ont,but seemingly not in the Spec doc. Will PR to fix:

* [SpatialMeasure in the ont](https://github.com/opengeospatial/ogc-geosparql/blob/master/1.1/geo.ttl#L511)

Reply:
Actually, this is solved in the Spec master version, see https://github.com/opengeospatial/ogc-geosparql/blob/master/1.1/spec/09-Part-06.adoc so I think this can be closed.